.github/workflows/pythontest.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       # Label used to access the service container
       postgres:
         # Docker Hub image
-        image: postgres
+        image: postgres:12.10
```

Comment:
I think it would be okay to simply have the versions be the major version `12`. Any reason why you chose to include the minor version?

Reply:
I saw that previously we had postgres 9.6 pinned on docker compose, so I thought it would be good to pin the minor for `v12` too. I am fine with pinning just the major version though.